jsk_robot_common/jsk_robot_startup/util/mux_selector.py

```diff
@@ -19,13 +19,15 @@
 """
 
 import rospy
-import thread
 import sys
 import roslib.message
 import rostopic
 from topic_tools.srv import MuxSelect
 import traceback
-
+if sys.version.major == 2:
+    import thread
+else:
+    import _thread
```

Comment:
How about writing `import _thread` as `import _thread as thread`?

Reply:
Thank you for advice. I think that's a good idea.
Modified in [`642da39` (#1517)](https://github.com/jsk-ros-pkg/jsk_robot/pull/1517/commits/642da39f4a9f6e8298ced5e6cbbcfca0d9c7825f)

Reply:
sorry, I made a mistake.

Reply:
Following changes will be good.
```
import sys
if sys.version_info.major < 3:
    import thread
else:
    import _thread as thread
```

Reply:
Thanks. I fixed.